src/test/scala/chiselTests/LayerSpec.scala

```diff
@@ -160,6 +160,39 @@ class LayerSpec extends ChiselFlatSpec with Utils with MatchesAndOmits {
     )()
   }
 
+  they should "allow for defines to layer-colored probes without layer blocks" in {
+
+    class Foo extends RawModule {
+      val a, b = IO(Input(Bool()))
+      val x = IO(Output(Probe(Bool(), A)))
+      val y = IO(Output(Probe(Bool(), A.B)))
+      define(x, ProbeValue(a))
+      define(y, ProbeValue(b))
+    }
+
+    matchesAndOmits(ChiselStage.emitCHIRRTL(new Foo))(
+      "define x = probe(a)",
+      "define y = probe(b)"
+    )()
```

Comment:
`ChiselStage.elaborate` is sufficient to only check that it doesn't error.

Reply:
Good call, thanks! Replaced with `ChiselStage.convert`!